app/controllers/api/api_controller.rb

```diff
@@ -0,0 +1,23 @@
+class Api::ApiController < Api::V1::BaseApiController
+  def index
+    json = Rails.cache.fetch('API_JSON_RESPONSE', expires_in: 1.day) do
```

Comment:
Probably not necessary, but I figure best to cache whenever the contents aren't likely to change.

I thought we had added a cache busting step as part of the deploy process. Can't find mention of it in Chef recipes, so that may have not happened. Something to keep in mind for the future.